src/LiveObjects.h

```diff
@@ -76,14 +77,16 @@ enum LiveObjects_variableType {
 enum Protocol
 {
   MQTT,
-  //,SMS
+  SMS
   //,LORA
 };
 
-enum Security
+enum Mode
 {
   NONE
   ,TLS
+  ,BIN_
+  ,TXT

```

Comment:
consider changing "BIN_" maybe to "BINR" ?

Reply:
Is it still relevant regarding to the new SMS 'protocol'? Perhaps more accurate to have begin(protocol, debug) with protocol=MQTT, SMS, LORA, and use setSecurity in conjunction with MQTT only (when needed).
+ additional comments on SMS types by email, to come.